src/components/OrgFile/OrgFile.unit.test.js

```diff
@@ -89,57 +91,57 @@ describe('Unit Tests for Org file', () => {
       return expect(strippedDescription);
     };
 
-    test('Parse simple description', () => {
+    test('Parse empty description', () => {
```

Comment:
Did the linter point you to this??

Reply:
Yes; if you review commit by commit then you'll see that this change came from commit a497c493d98d0233e668e487e809e6320e74a25f which has the title `eslint: Fix jest/no-identical-title rule`.  If that's not entirely self-explanatory then I apologise for having been too lazy to write a proper commit message yet :wink:  Although in my defense, this is still a draft PR :stuck_out_tongue_winking_eye:

Reply:
I've now added commit messages and force-pushed, so the new version of the above commit is d17230f.